utils/BaseStore.js

```diff
@@ -1 +1,4 @@
-module.exports = require('dispatchr/utils/BaseStore');
+// @TODO deprecate
+//console.warn("require('fluxible/utils/createStore') is deprecated. Please use " +
```

Comment:
:+1: on doing it with a Github milestone. Thanks for being considerate to library users.